README.md

```diff
@@ -10,22 +10,28 @@
-| WORLD                | `Dedicated`            | TRUE     | This is used to generate the name of your world. |
-| PUBLIC               | `1`                    | FALSE    | Sets whether or not your server is public on the server list. |
-| PASSWORD             | `12345`                | TRUE     | Set this to something unique! |
-| AUTO_UPDATE          | `0`                    | FALSE    | Set to `1` if you want your container to auto update! This means at 1 am it will update, stop, and then restart your server. |
-| AUTO_UPDATE_SCHEDULE | `0 1 * * *`            | FALSE    | This works in conjunction with `AUTO_UPDATE` and sets the schedule to which it will run an auto update. [If you need help figuring out a cron schedule click here](https://crontab.guru/#0_1_*_*_*) |
+| Variable                 | Default                | Required | Description |
+|--------------------------|------------------------|----------|-------------|
+| TZ                       | `America/Los_Angeles`  | FALSE    | Sets what timezone your container is running on. This is used for timestamps and cron jobs. [Click Here for which timezones are valid.](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) |
+| PUID                     | `1000`                 | FALSE    | Sets the User Id of the steam user. |
+| PGID                     | `1000`                 | FALSE    | Sets the Group Id of the steam user. |
+| PORT                     | `2456`                 | TRUE     | Sets the port your server will listen on. Take not it will also listen on +2 (ex: 2456, 2457, 2458) |
+| NAME                     | `Valheim Docker`       | TRUE     | The name of your server! Make it fun and unique! |
+| WORLD                    | `Dedicated`            | TRUE     | This is used to generate the name of your world. |
+| PUBLIC                   | `1`                    | FALSE    | Sets whether or not your server is public on the server list. |
+| PASSWORD                 | `12345`                | TRUE     | Set this to something unique! |
+| AUTO_UPDATE              | `0`                    | FALSE    | Set to `1` if you want your container to auto update! This means at 1 am it will update, stop, and then restart your server. |
+| AUTO_UPDATE_SCHEDULE     | `0 1 * * *`            | FALSE    | This works in conjunction with `AUTO_UPDATE` and sets the schedule to which it will run an auto update. [If you need help figuring out a cron schedule click here]
+| AUTO_BACKUP              | `0`                    | FALSE    | Set to `1` to enable auto backups. Backups are stored under `/home/steam/backups` which means you will have to add a volume mount for this directory. |
+| AUTO_BACKUP_SCHEDULE     | `*/15 * * * *`         | FALSE    | Change this value to update based on how frequently you would like the server to update. [If you need help figuring out a cron schedule click here].
```

Comment:
"how frequently you would like the server to update." should probably be "how frequently you would like the server to backup.